packages/app/email-notifications/__tests__/email-notifications.test.ts

```diff
@@ -132,7 +132,7 @@ describe('Email Notifications', () => {
         ${'Init'}         | ${''}             | ${'FAQs'}   | ${modifyReviewPublishWorkflow} | ${[]}
         ${'Draft'}        | ${'Init'}         | ${'FAQs'}   | ${modifyReviewPublishWorkflow} | ${[]}
         ${'Under Review'} | ${'Draft'}        | ${'FAQs'}   | ${modifyReviewPublishWorkflow} | ${['eggs', 'hashbrowns']}
-        ${'Draft'}        | ${'Under Review'} | ${'FAQs'}   | ${modifyReviewPublishWorkflow} | ${['bacon']}
+        ${'Draft'}        | ${'Under Review'} | ${'FAQs'}   | ${modifyReviewPublishWorkflow} | ${['eggs', 'hashbrowns']}
```

Comment:
Ah, I think it finally got it—thanks Krupa! So before, bacon was notified because that user, as the author, was the only user able to transition the document through the edge by "Submit for review". Now because of `getTargetUserRoles` in the workflow service, we notify any users matching the edge's `notifyRoles` value (`Reviewer`, in this case) for the FAQs model...same as with the changed snapshot, above.

Reply:
Yes that is correct.